pymess/models/emails.py

```diff
@@ -12,6 +12,8 @@
 from django.template import Template, Context
 from django.template.exceptions import TemplateSyntaxError, TemplateDoesNotExist
 
+from mimetypes import guess_type
```

Comment:
mimetypes is not in install_requires.You cannot use it here

Reply:
better will be add content_type to the EmailTemplateStaticAttachment and guess_type in the admin form